site-src/implementation-table.md

```diff
@@ -0,0 +1,13 @@
+| istio                         | TLS MESH HTTP      | 1.2           | ['HTTPRoutePathRedirect', 'HTTPRouteRequestMirror', 'HTTPRouteBackendTimeout', 'HTTPRouteResponseHeaderModification', 'HTTPRoutePortRedirect', 'HTTPRouteSchemeRedirect', 'HTTPRoutePathRewrite', 'HTTPRouteRequestTimeout', 'HTTPRouteQueryParamMatching', 'HTTPRouteRequestMultipleMirrors', 'HTTPRouteMethodMatching', 'HTTPRouteHostRewrite']                           | ['HTTPRouteRequestTimeout', 'HTTPRoutePathRedirect', 'HTTPRouteRequestMirror', 'HTTPRoutePathRewrite', 'HTTPRouteMethodMatching', 'HTTPRouteRequestMultipleMirrors', 'HTTPRouteBackendTimeout', 'HTTPRouteResponseHeaderModification', 'HTTPRoutePortRedirect', 'HTTPRouteSchemeRedirect', 'HTTPRouteHostRewrite', 'HTTPRouteQueryParamMatching'] |
+| kubernetes-ingress-controller | HTTP HTTP          | v3.1.1        | ['HTTPRouteMethodMatching', 'HTTPRouteBackendTimeout', 'HTTPRouteQueryParamMatching', 'HTTPRouteResponseHeaderModification']                                                                                                                                                                                                                                                | N/A                                                                                                                                                                                                                                                                                                                                               |
+| kubernetes-ingress-controller | HTTP HTTP          | v3.0.2        | ['HTTPRouteMethodMatching', 'HTTPRouteQueryParamMatching', 'HTTPRouteResponseHeaderModification']                                                                                                                                                                                                                                                                           | N/A                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
This one is kong-kubernetes-ingress-controller, Probably the logic removes the org name somehow. Other than that, the only profile (HTTP) is duplicated; my guess is that the kong ingress controller is the only implementation to have multiple reports so far, so it may be the profiles get duplicated somehow.

Reply:
good catch! the pr that formalized the reporting procedure was merged, making some changes to address them  😁